components/homme/test_execs/theta-l-nlev128-native/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+#_______________________________________________________________________
+# create executable needed by planar density current (native output)
```

Comment:
there is already theta exec with 128 levels, theta-nlev128 .

aslo, it may be too much work here but would it be good to get rid of -l- suffix in theta-l?

Reply:
I introduced this because the planar code doesn't yet support interpolated (non-native) grid output, so I need theta execs that use native output.

I agree that we should change theta-l to theta at some point, but the name theta-l is prevalent throughout the code, so this should probably be a separate pull request.